.github/release.yml

```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - octocat
+      - dependabot
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - Semver-Major
+        - breaking-change
```

Comment:
do you think that we will actually use these labels? I am a bit skeptical.

Reply:
Also skeptical, that's just what we have in `spaghetti`. It's the boiler plate from the action template.

Reply:
I'd try to adapt it a bit to our use case. So removing `Semver-*` which I doubt anyone will ever use. We can see how the rest will work.